state/mongodb/mongodb.go

```diff
@@ -167,6 +167,11 @@ func (m *MongoDB) Get(req *state.GetRequest) (*state.GetResponse, error) {
 	filter := bson.M{id: req.Key}
 	err := m.collection.FindOne(ctx, filter).Decode(&result)
 	if err != nil {
+		if err.Error() == "mongo: no documents in result" {
```

Comment:
@yaron2 @artursouza Isn't this a breaking change? 
My question is do we want to change it this way now? 
Should the code be changed specifically for conf-tests? Or should it be handled as a special case in conf-tests?

Reply:
If this is a breaking change for the component, this is all the more reason to do this now.

Reply:
We better break it now then later.